modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/RichTextTest.php

```diff
@@ -108,7 +108,7 @@ public function testRendering(): void {
       'second option',
```

Comment:
add 
```
/**
   * {@inheritdoc}
   */
  protected static $modules = [
    'twig_field_value',
  ];
```
the reason is that only this one needs it, lets start to be more efficient, the tests are slow enough